styles.css

```diff
@@ -89,11 +122,635 @@
+}
+
+.card-button:hover {
+  border: 1px solid #f55800;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.about-section {
+  background: #1c1a19;
+  padding: 100px 25px 0;
+}
+
+.about-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 42px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-text {
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+  color: #f4f5f7;
+}
+
+.about-button {
+  background: #ff6b00;
+  padding: 12px;
+  font-size: 15px;
+  margin-top: 20px;
+}
+
+.about-button:enabled {
+  background: #f55800;
+  color: #fff;
+  border: none;
+}
+
+.about-button:disabled {
+  background: #e5e5e5;
+  border: none;
+  color: #5b5957;
+}
+
+.about-button:hover {
+  border: 1px solid #f55800;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.about {
+  background-image: url("./images/about\ image\ .png");
+  background-repeat: no-repeat;
+  background-position: right 0% bottom 0%;
+  background-size: 120px;
+  height: 70vh;
+}
+
+.language {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+}
+
+.language-list {
+  margin: 10px auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 43px;
+  text-transform: capitalize;
+}
+
+.img {
+  margin-right: 8px;
+}
+
+.intro-text {
+  width: 250px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 23px;
+  line-height: 39px;
+  text-align: center;
+  color: #172b4d;
+  margin-left: 5rem;
+  margin-top: 30px;
+}
+
+.form {
+  font-family: sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 58px 0;
+}
+
+.get-button {
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  width: 131px;
+  padding: 12px;
+  font-size: 15px;
+  margin: 10px 1px;
+}
+
+.get-button:hover {
+  border: 1px solid;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.get-button:disabled {
+  background: #e5e5e5;
+  border: none;
+  color: #5b5957;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: space-evenly;
+  padding-top: 10px;
+  max-width: 100px;
+  margin: 70px 0 110px;
+  margin-left: 200px;
+}
+
+.social-logos {
+  color: #505f79;
+  width: 15px;
+  height: 15px;
+  margin-top: 10px;
+}
+
+#fname,
+#email {
+  border-bottom: 1px solid #dbdbd7;
+  border-right-width: 0;
+  border-top-width: 0;
+  border-left-width: 0;
+  padding: 15px 0 15px 15px;
+  margin-top: 12px;
+  width: 327px;
+}
+
+.textarea {
+  width: 327px;
+  height: 114px;
+  background-color: #fbf8f7;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  padding-top: 12px;
+  font-family: 'inter', sans-serif;
+  padding-left: 12px;
+  margin-top: 20px;
+}
+
+.nav-list {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .grid-container {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    margin: 50px;
+  }
```

Comment:
Your mobile design no longer looks like the one from the template design.

<img width="247" alt="Screen Shot 2022-05-30 at 15 23 29" src="https://user-images.githubusercontent.com/82426682/171055570-5ac4b54a-5b12-4a09-9adf-be103dfb91c8.png">

There is now a strange margin to the right of the design. Kindly fix this as your mobile design shouldn't have changed when implementing the styles for the desktop version.